### ISSUE TITLE
Issue of losing data in staging

### ISSUE THREAD

**Issue by samvthom16** [akvo/Akvo-Theme#198]
Answer the following questions: 
 - When did this happen?
 - How many times did the issue repeat?
 - Could you explain step by step on how you faced the issue? (Would help to replicate)

**Comment by Lindaleunissen** [comment]
I made some changes to the Europe page last week. New header image, new map marker, possibly some font changes, though I cannot be sure of that. I think they were still there on Monday, but on Tuesday I noticed they were replaced by the previous images. 

I haven't made changes to other pages, so don't know if it happened there too. 

On the Europe page there were also some changes in Chris' code. Things that were working correctly before, were not working now.
